Userland/mount.cpp

```diff
@@ -96,11 +96,20 @@ bool mount_all()
             continue;
         }
 
-        dbg() << "Mounting " << devname << "(" << fstype << ")"
+        const char* filename = parts[0].characters();
+        int fd = open(filename, O_RDONLY);
+        if (fd < 0) {
+            int saved_errno = errno;
+            auto message = String::format("Failed to open : %s", filename);
```

Comment:
Space before the colon, not cool :slightly_frowning_face:

Reply:
Oops, my bad!